app/views/decidim/accountability/results/_header.html.erb

```diff
@@ -0,0 +1,6 @@
+
+<p>Te contamos el progreso del Plan de Actuación Municipal ...</p>
```

Comment:
@andreslucena thanks, I know, this PR only changes the structure and puts the elements of the design made by @furilo in the views, so @martgnz can then add the right html markup and css for the new design, this is only a placeholder text while things are in development.

Also in this case it would probably have to be a text that's stored somehow in the db, as it probably will be different for each process that uses the accountability module, and there could be many of those. Unless you have a standard text that could work for any process just by changing the name of the process there. If you have a standard text please let me know and I will add just that in a translation string.